bin/latexmlpost

```diff
@@ -289,12 +290,14 @@ else {
 if ((!defined $destination)
   && ($dographics || $picimages
     || checkMathFormat('images') || checkMathFormat('svg'))) {
-  print STDERR "WARNING: must supply destination to support auxilliary files;\n"
-    . "  disabling: --nomathimages --nographicimages --nopictureimages\n";
+  print STDERR generateMessage(colorizeString("Warning:expected:options", 'warning'), undef,
```

Comment:
a bit iffy to have to unpack the body of `Warn()` here, but since `Warn()` depends on `$STATE` I decided that is an unnecessary complication to avoid. I basically needed a stateless Warn, could refactor it to a new subroutine e.g. `WarnExt` for "external" to the core conversion warning?

Reply:
The reason I wanted a real warning was to get the color highlight, since it's very easy to miss in the regular flow of conversion text.